boards/nucleo_f429zi/src/main.rs

```diff
@@ -258,132 +258,132 @@ pub unsafe fn reset_handler() {
-                                                      // stm32f4xx::gpio::PIN[5][10].as_ref().unwrap(), //A5
-                                                      // stm32f4xx::gpio::PIN[1][1].as_ref().unwrap(), //A6
-                                                      // stm32f4xx::gpio::PIN[2][2].as_ref().unwrap(), //A7
-                                                      // stm32f4xx::gpio::PIN[5][4].as_ref().unwrap()  //A8
+        stm32f429zi::gpio::PIN[3][7].as_ref().unwrap(), //D51
+        stm32f429zi::gpio::PIN[3][6].as_ref().unwrap(), //D52
+        stm32f429zi::gpio::PIN[3][5].as_ref().unwrap(), //D53
+        stm32f429zi::gpio::PIN[3][4].as_ref().unwrap(), //D54
+        stm32f429zi::gpio::PIN[3][3].as_ref().unwrap(), //D55
+        stm32f429zi::gpio::PIN[4][2].as_ref().unwrap(), //D56
+        stm32f429zi::gpio::PIN[4][4].as_ref().unwrap(), //D57
+        stm32f429zi::gpio::PIN[4][5].as_ref().unwrap(), //D58
+        stm32f429zi::gpio::PIN[4][6].as_ref().unwrap(), //D59
+        stm32f429zi::gpio::PIN[4][3].as_ref().unwrap(), //D60
+        stm32f429zi::gpio::PIN[5][8].as_ref().unwrap(), //D61
+        stm32f429zi::gpio::PIN[5][7].as_ref().unwrap(), //D62
+        stm32f429zi::gpio::PIN[5][9].as_ref().unwrap(), //D63
+        stm32f429zi::gpio::PIN[6][1].as_ref().unwrap(), //D64
+        stm32f429zi::gpio::PIN[6][0].as_ref().unwrap(), //D65
+        stm32f429zi::gpio::PIN[3][1].as_ref().unwrap(), //D66
+        stm32f429zi::gpio::PIN[3][0].as_ref().unwrap(), //D67
+        stm32f429zi::gpio::PIN[5][0].as_ref().unwrap(), //D68
+        stm32f429zi::gpio::PIN[5][1].as_ref().unwrap(), //D69
+        stm32f429zi::gpio::PIN[5][2].as_ref().unwrap(), //D70
+        stm32f429zi::gpio::PIN[0][7].as_ref().unwrap()  //D71
+
+                                                        // ADC Pins
+                                                        // Enable the to use the ADC pins as GPIO
+                                                        // stm32f429zi::gpio::PIN[0][3].as_ref().unwrap(), //A0
+                                                        // stm32f429zi::gpio::PIN[2][0].as_ref().unwrap(), //A1
+                                                        // stm32f429zi::gpio::PIN[2][3].as_ref().unwrap(), //A2
+                                                        // stm32f429zi::gpio::PIN[5][3].as_ref().unwrap(), //A3
+                                                        // stm32f429zi::gpio::PIN[5][5].as_ref().unwrap(), //A4
+                                                        // stm32f429zi::gpio::PIN[5][10].as_ref().unwrap(), //A5
+                                                        // stm32f429zi::gpio::PIN[1][1].as_ref().unwrap(), //A6
+                                                        // stm32f429zi::gpio::PIN[2][2].as_ref().unwrap(), //A7
+                                                        // stm32f429zi::gpio::PIN[5][4].as_ref().unwrap()  //A8
```

Comment:
I'm wondering what's happening with the formatting here. Maybe a block comment would be better than line-by-line comments?

Reply:
Rustfmt seems to handle this case really poorly, even with block comments. I moved the commented out pins to the top of the argument list instead.